packages/opds-web-client/src/components/Book.tsx

```diff
@@ -237,6 +252,41 @@ export default class Book<P extends BookProps> extends React.Component<P, void>
     return links;
   }
 
+  getMedium(book) {
+    if (!book.raw || !book.raw["$"] || !book.raw["$"]["schema:additionalType"]) {
+      return "";
+    }
+
+    return book.raw["$"]["schema:additionalType"].value ?
+      book.raw["$"]["schema:additionalType"].value : "";
+  }
+
+  getMediumSVG(medium, displayLabel = true) {
+    if (!medium) {
+      return null;
+    }
+
+    const svgMediumTypes = {
+      "http://bib.schema.org/Audiobook": {
+        element: <AudioHeadphoneIcon ariaHidden title="Audio/Headphone SVG Icon" />,
```

Comment:
I'm curious why "SVG" is part of the title?

Reply:
No real reason and it can be removed. It was just the convention we used here but we don't need to point out that it's SVG to screenreaders, especially if we're hiding it.